dags/locations/__init__.py

```diff
@@ -20,11 +20,11 @@
         "slack": dagster_slack.SlackResource(token=dagster.EnvVar("SLACK_TOKEN")),
         # Postgres resource (universal for all dags)
         "database": PostgresResource(
-            host=dagster.EnvVar("POSTGRES_HOST"),
-            port=dagster.EnvVar("POSTGRES_PORT"),
-            database=dagster.EnvVar("POSTGRES_DATABASE"),
-            user=dagster.EnvVar("POSTGRES_USER"),
-            password=dagster.EnvVar("POSTGRES_PASSWORD"),
+            host=dagster.EnvVar("PGHOST"),
+            port=dagster.EnvVar("PGPORT"),
+            database=dagster.EnvVar("PGDATABASE"),
+            user=dagster.EnvVar("PGUSER"),
+            password=dagster.EnvVar("PGPASSWORD"),
```

Comment:
The EnvVar here will get the value, so there is no need for it to also be on the `PostgresResource`